tests/test_api.py

```diff
@@ -76,7 +76,7 @@ async def check_camera(camera: Camera):
 
     camera.last_ring_event  # noqa: B018
 
-    assert camera.timelapse_url == f"https://127.0.0.1/protect/timelapse/{camera.id}"
+    assert camera.timelapse_url == f"https://127.0.0.1:0/protect/timelapse/{camera.id}"
```

Comment:
0 isn't a default port for http so it should be showing up.

This has been flipping back and forth in yarl releases. We need tests for port 0 upstream so the behavior stays the same.  Its currently correct in 1.14.0